### PR TITLE
Feature/navbar unit tests

### PR DESCRIPTION
**Pull request by tomvik**
### What does this PR do?

- Fixes eslint and flow errors on Navbar
- Adds unit tests to Navbar
- Fixes flow errors in App.test.js
- Updates jest and testing dependencies
- Configures jest and babel to support async

#### Related issues

#29

#### Merge Checklist:

- [x] The branch is up-to-date (i.e. rebased) with the base branch
- [x] No code style issues (eslint and flow)

### CODE REVIEW

**Comment by tomvik** [comment]
Hi @RicardoChapaRomero this should be ready for review. Thanks!

**Comment by tomvik** [comment]
friendly ping @RicardoChapaRomero

File: src/components/NavBar/NavBar.test.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        expect(current_button.className).not.toContain('active')
+      }
+    } else {
+      expect(current_button).not.toEqual(null)
+    }
+  })
+})
+
+it('<NavBar> Links correctly when clicked', () => {
+  const history = createMemoryHistory()
```
Comment (RicardoChapaRomero) [inline src/components/NavBar/NavBar.test.js:69]: Can you leave a comment explaining the use of `createMemoryHistory()`

**Review by RicardoChapaRomero — APPROVED** [review]
Good work Tommy! LGTM

**Pull request merged**